quickstarts/dotnet/restsharp/config.yml

```diff
@@ -2,20 +2,24 @@ id: 4967824f-1a29-417f-88f2-093d30a41fc2
 name: restsharp
 description: |
   ## What is RestSharp?
+  
+  RestSharp is a popular REST API client library for [.NET](https://newrelic.com/instant-observability/dotnet/2dff13b6-0fac-43a6-abc6-57f0a3299639) for the purpose of making synchronous and asynchronous calls to remote resources over HTTP. Check out [.NET documentation](https://docs.newrelic.com/docs/apm/agents/net-agent/installation/introduction-net-agent-install/) for more.
```

Comment:
```suggestion
  RestSharp is a popular REST API client library for .NET with the purpose of making synchronous and asynchronous calls to remote resources over HTTP. Check out our [.NET documentation](https://docs.newrelic.com/docs/apm/agents/net-agent/installation/introduction-net-agent-install/) for more information.
```

Reply:
This documentation link doesn't have any references to Restsharp but it seems our .NET agent supports it.

https://docs.newrelic.com/docs/release-notes/agent-release-notes/net-release-notes/net-agent-6173870/

Reply:
I removed this link to the .NET quickstart as I felt it would a little confusing to point a user who is installing this quickstart to another quickstart.